doc/tech/usecases.md

```diff
@@ -11,3 +11,12 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+
+Content 
+•	Objective ( High level Objective ) 
+•	Scope ( focus on Edge Scope , in/out scope , Next Slide ) 
```

Comment:
Next slide?

Reply:
it is Just draft point that we need to work on it

Reply:
Still I think we should no have next slide here unless we refer to the slideset.

Reply:
Thanks, I already removed it after your first comment :)